File: engine/engine-model/src/main/scala/net/server/channel/packet/alliance/DenyAllianceRequestPacket.scala
package net.server.channel.packet.alliance

import net.server.MaplePacket

class DenyAllianceRequestPacket(private var _inviterName: String, private var _guildName: String) extends MaplePacket {
     def inviterName: String = _inviterName
     def guildName: String = _guildName
}

File: engine/engine-model/src/main/java/net/server/channel/packet/alliance/DenyAllianceRequestPacket.java
package net.server.channel.packet.alliance;

public class DenyAllianceRequestPacket extends AllianceOperationPacket {
   private final String inviterName;

   private final String guildName;

   public DenyAllianceRequestPacket(String inviterName, String guildName) {
      this.inviterName = inviterName;
      this.guildName = guildName;
   }

   public String inviterName() {
      return inviterName;
   }

   public String guildName() {
      return guildName;
   }
}
